build-tools/packages/version-tools/src/virtualPatchScheme.ts

```diff
@@ -0,0 +1,123 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import * as semver from "semver";
+import { VersionBumpType } from "./bumpTypes";
+
+/**
+ * Determines if a version is a virtual patch format or not using a very simplistic algorithm.
+ */
+export function isVirtualPatch(version: semver.SemVer | string): boolean {
+    // If the major is 0 and the patch is >= 1000 assume it's a virtualPatch version
+    if (semver.major(version) === 0 && semver.patch(version) >= 1000) {
```

Comment:
`1000` appears in a lot of places. Should we define a constant for it somewhere and document it?